src/sentry_log.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+
+#include "sentry_log.h"
+#include "sentry_options.h"
+
+#if defined(SENTRY_PLATFORM_ANDROID)
+
+#    include <android/log.h>
+void
+sentry_log_hook_default(sentry_level_t level, const char *message, va_list args)
+{
+    __android_log_vprint(ANDROID_LOG_INFO, "sentry-native", message, args);
+}
+
+#else
+
+void
+sentry_log_hook_default(sentry_level_t level, const char *message, va_list args)
+{
+    vfprintf(stderr, message, args);
+}
+
+#endif
+
+void
+sentry_log(sentry_level_t level, const char *message, ...)
```

Comment:
In general, we would like to prefix internal functions with double underscores, like this:

```suggestion
sentry__log(sentry_level_t level, const char *message, ...)
```

The same is true for the default function above.